Tests/YamsTests/ConstructorTests.swift

```diff
@@ -210,14 +210,16 @@ class ConstructorTests: XCTestCase { // swiftlint:disable:this type_body_length
             english: null
             ~: null key
             ---
-            # This sequence has five
-            # entries, two have values.
+            # This sequence has seven
+            # entries, four have values.
             sparse:
               - ~
               - 2nd entry
               -
               - 4th entry
               - Null
+              - 'null'
+              - ''
```

Comment:
Before the change to `NSNull.construct(yaml:)`, these tests passed anyway. This is because of other stuff that is done when calling `Yams.load_all(yaml:)`. I just made the changes here anyway to highlight the expectation incase things change in the future.